src/rpc/misc.cpp

```diff
@@ -25,6 +25,8 @@
 
 static UniValue validateaddress(const JSONRPCRequest& request)
 {
+    const std::string example_address = "\"bc1q09vm5lfy0j5reeulh4x5752q25uqqvz34hufdl\"";
```

Comment:
Compile-time constants are usually all-upper-case. Also, I wouldn't mind to have a single place where this is defined/declared instead of one for each RPC that uses it.

Reply:
Good idea. Perhaps like `UNIX_EPOCH_TIME` in src/rpc/util.h and util.cpp in e2f32cb

Reply:
In this case it might be good to update doc/developer-notes.md as well, see the last two entries about bech32 addresses and UNIX epoch time.

Reply:
@MarcoFalke @jonatack: Thanks for reviewing! I pushed a commit that factors out `EXAMPLE_ADDRESS` into `src/rpc/util.{h,cpp}`. Will update the developer notes as well.